index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+const app = express()
+
+app.listen(3001, ()=> {
+  console.log('app listening on 3k1')
+})
+
+app.get('/:name', (req, res) => {
+  res.send(`hello,  ${req.params.name}`)
+})
```

Comment:
Let me know if you'd like some help figuring out how to implement the random color and/or compliment feature.